source/Account.cpp

```diff
@@ -369,12 +369,17 @@ const vector<Mortgage> &Account::Mortgages() const
 
 
 
-// Add a new mortgage for the given amount, with an interest rate determined by
-// your credit score.
-void Account::AddMortgage(int64_t principal)
+// Add a new mortgage. If no interest rate is given then the player's credit score
+// is used to determine the interest rate. If addCredits is true then the principal
+// is also added to the player's credits; otherwise, this is just debt.
+void Account::AddMortgage(int64_t principal, optional<double> interest, int term, bool addCredits)
```

Comment:
Ok don't hate me, but since the mortgage node isn't actually adding credits, it does seem like a good idea to separate it to a new function called say `AddDebt`  instead of adding a bool parameter to control what the function does.

Reply:
🙃

Reply:
Went an extra mile.